nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/resources/test-operations.xml

```diff
@@ -122,6 +122,16 @@
       </operation>
     </chain>
 
+    <chain id="testAddPermissionToUsers">
+      <operation id="Document.AddPermission">
+        <param type="string" name="permission">Read</param>
+        <param type="string" name="acl">local</param>
+        <param type="boolean" name="blockInheritance">false</param>
+        <param type="boolean" name="notify">false</param>
```

Comment:
The last 3 can be removed as they are the default values.

Reply:
I wanted to do it before, but i keeped it (to have overview and then discuss about it:)). because i was wondering how i can check that the params values was correctly saved but i wasn't able to get this informations from ACE type. org.nuxeo.ecm.core.api.security.ACE#contextData is empty and when debuging i was able to see that the org.nuxeo.ecm.core.api.security.ACE#contextData was save with notify to false value by example

Reply:
The ACE's contextData isn't be stored when setting it on the DocumentModel with `doc.setACP()`, that's why it is null.
Yet, if you really wanted to test these you could use the `CapturingEventListener`.

Reply:
I agree, remove the unnecessary parameters here to show what's really tested: add the `READ` permission to `jchastain, rdeniro, dwashington`.

Reply:
yep,